README.md

```diff
@@ -96,196 +44,117 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- **create database and table for animals**
+- **insert animals data into the table**
+- **make query for the table data**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
+- [PostgreSQL database server](https://www.postgresql.org/download/) installed
 
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+git clone git@github.com:Gmanlove/Vet-clinic-database.git
 
-### Install
+cd Vet-clinic-database
 
-Install this project with:
+### Create database
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
+1. Connect to your PostgreSQL server with `psql`
 ```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
+> psql
+postgres=#
 ```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
 
-```sh
-  bin/rails test test/models/article_test.rb
+2. Create the database `vet_clinic`
+```
+postgres=# CREATE DATABASE vet_clinic;
+CREATE DATABASE
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
 
-<!--
-Example:
+3. Connect to your database vet_clinic. Inside your current session do:
+```
+postgres=# \c vet_clinic
+You are now connected to database "vet_clinic" as user "postgres".
+vet_clinic=#
+```
 
-```sh
+That's it! Congratulations, you have created your database and connected to it. Next, we will add a table.
 
-```
- -->
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
+👤 **Emmanuel Obi**
 
```

Comment:
- [ ] [Optional] [Highly recommended] I can see that you use two different GitHub accounts to push your data.
Kindly next time use one account this one @Gmanlove because this is the official account linked with Microverse.
Because if you did this again will bring confusion to the reviewers reviewing your projects.

Reply:
Thanks... please can u assist me solve this issue.

Reply:
sure If you are free now contact me on slack